src/ciphersuite/mod.rs

```diff
@@ -7,7 +7,11 @@ use log::error;
 
 use evercrypt::prelude::*;
 use hpke::prelude::*;
-use serde::{Deserialize, Serialize};
+use serde::{
```

Comment:
We should `pub(crate) use` these somewhere centrally to not having to include all of this every time.